shop/src/main/java/ro/msg/learning/shop/service/LocationService.java

```diff
@@ -0,0 +1,15 @@
+package ro.msg.learning.shop.service;
+
+import ro.msg.learning.shop.dto.LocationDto;
+import ro.msg.learning.shop.exception.LocationNotFoundException;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface LocationService {
```

Comment:
We can use interface sure. It is not a wrong idea but in this case we need an interface for every service and we do not benefit from it. I think you can leave only the implementations for this app.